TabulateSmarterTestAdminPackage.Common/Validators/MinIntValueValidator.cs

```diff
@@ -0,0 +1,28 @@
+﻿using TabulateSmarterTestAdminPackage.Common.Enums;
+
+namespace TabulateSmarterTestAdminPackage.Common.Validators
+{
+    public class MinIntValueValidator : Validator
+    {
+        public MinIntValueValidator(ErrorSeverity errorSeverity, object parameter) : base(errorSeverity, parameter)
+        {}
+
+        public override bool IsValid(object value)
+        {
+            int intValue;
+            int intParameter;
+            var valueString = value as string;
+            var parameterString = Parameter as string;
+            return !string.IsNullOrEmpty(valueString)
```

Comment:
Devs have strong opinions on this.  Personally I prefer `String.IsNullOrEmpty()` and `Int32.TryParse` (the type name instead of the alias) when calling static methods and the alias (lower case) when using it as the variable.

I'm fine keeping it this way if the existing code used that as a standard.  You let me know.